docs/docs/misc/migration_notes.md

```diff
@@ -6,6 +6,28 @@ keywords: [sandbox, cli, aztec, notes, migration, updating, upgrading]
 
 Aztec is in full-speed development. Literally every version breaks compatibility with the previous ones. This page attempts to target errors and difficulties you might encounter when upgrading, and how to resolve them.
 
+## [T.B.D]
```

Comment:
I think the next release will be 0.36.0. Rahul had me rename the TBD to that but double check with him.